backend/src/common/client/api_server/experiment_client_fake.go

```diff
@@ -9,7 +23,6 @@ import (
 )
 
 const (
-	ExperimentForDefaultTest     = "EXPERIMENT_DEFAULT"
```

Comment:
To confirm: was this an unused constant?

Reply:
correct